tests/components/homekit_controller/specific_devices/test_lg_tv.py

```diff
@@ -29,8 +33,22 @@
     # Assert that the friendly name is detected correctly
     assert state.attributes["friendly_name"] == "LG webOS TV AF80"
 
+    # Assert that all channels were found and that we know which is active.
+    assert state.attributes["source_list"] == [
+        "AirPlay",
+        "Live TV",
+        "HDMI 1",
+        "Sony",
+        "Apple",
+        "AV",
+        "HDMI 4",
+    ]
+    assert state.attributes["source"] == "HDMI 4"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5141152)